src/Api/Controllers/CollectionsController.cs

```diff
@@ -232,16 +246,38 @@ public async Task Delete(Guid orgId, Guid id)
 
     [HttpDelete("")]
     [HttpPost("delete")]
-    public async Task DeleteMany([FromBody] CollectionBulkDeleteRequestModel model)
+    public async Task DeleteMany(Guid orgId, [FromBody] CollectionBulkDeleteRequestModel model)
```

Comment:
That property has already been removed in the feature branch (on both clients and server), because it's not required for the auth handler interface. Rather than restore it for the legacy logic, and then have to remove it again later, we can just grab the orgId from the URL and use that instead.